activate_dev_from_wheel.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Python environment name
+PYTHON_ENV_NAME=$1
+
+if [ -z "${PYTHON_ENV_NAME}" ]; then
+    echo "Error: Please provide a name for the Python virtual environment"
+else
+    export CATALYST_PYTHON_ENV_DIR=/tmp/$PYTHON_ENV_NAME
+
+    # We call the setup script in a sub-shell session, so in case of an error, the 
+    # current session does not get closed.
+    bash setup_dev_from_wheel.sh $PYTHON_ENV_NAME
+
+    # Activate the Python virtual environment session
+    python3 -m venv $CATALYST_PYTHON_ENV_DIR
+    source $CATALYST_PYTHON_ENV_DIR/bin/activate
```

Comment:
Importing the just installed package I get a warning I don't get in my regular Catalyst install:
```
/tmp/myenv/lib/python3.11/site-packages/requests/__init__.py:113: RequestsDependencyWarning: urllib3 (2.2.3.dev27) or chardet (None)/charset_normalizer (3.3.2) doesn't match a supported version!
  warnings.warn(
```
It's probably not a big deal but just thought I'd report it.

Reply:
Fixed by installing requirements first. Thanks!